src/CoinParamUtils.cpp

```diff
@@ -129,6 +130,129 @@ std::string nextField(const char *prompt)
 
 namespace CoinParamUtils {
 
+std::string printString(const std::string input, int maxWidth)
```

Comment:
input should be const std::string&, no need copy the whole string

Reply:
Yes, thanks.